src/components/shared/Validations.tsx

```diff
@@ -18,9 +17,9 @@ export const Validations: React.FunctionComponent<IValidations> = ({
   const validationCount = Object.keys(validations).length;
 
   const requiredElem = (
-    <div className={cn('ml-2', required ? 'font-medium' : 'text-darken-7 dark:text-lighten-6')}>
-      {required ? 'required' : 'optional'}
-      {validationCount ? `+${validationCount}` : ''}
+    <div>
+      {required && <span className="ml-2 text-orange-7 dark:text-lighten-6">required</span>}
```

Comment:
![image](https://user-images.githubusercontent.com/9273484/98110416-57680400-1e9f-11eb-882e-24566ce9eeb3.png)
```suggestion
      {required && <span className="ml-2 text-orange-7 dark:text-orange-6">required</span>}
```

Reply:
uups, didn't notice that.

Fixed, thanks for catching it :)